src/models/formItemInput.ts

```diff
@@ -0,0 +1,4 @@
+export interface FormItemInput<Value> {
+  value?: Value
+  onChange?: (value: Value) => void
+}
```

Comment:
**question**: im guessing there's a similar antd type for this already. Possible to use _it_ instead, or worth having our own?

Reply:
From a clean perspective, I like to have our own. The way I look at it is, it is just 'coincidental' it happens to match antd format ;)